assets/search/components/SearchResultsBar/SearchResultsFiltersRow.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
-
 import {gettext, getCreatedSearchParamLabel} from 'utils';
 
 import {SearchResultTagList} from './SearchResultTagList';
 import {Tag} from 'components/Tag';
 
 import {IProps as IParentProps} from './SearchResultTagsList';
+import {setItemTypeFilter} from 'agenda/actions';
+import {clearQuickFilter} from 'search/actions';
+import {searchFilterSelector} from 'search/selectors';
+import {connect} from 'react-redux';
+import {agendaCoverageStatusFilter, getActiveFilterLabel} from 'agenda/components/AgendaCoverageExistsFilter';
+
+const IS_AGENDA = location.pathname.includes('/agenda');
```

Comment:
We've already talked that this is a low quality implementation that we'd normally not approve, but are going with it anyway because of effort that would be required to do a proper one. In such cases it'd be very useful to add a comment describing the situation and guidelines on how should we fix it when we have more time.